src/app/screens/createPassword/index.tsx

```diff
@@ -49,20 +46,19 @@ function CreatePassword(): JSX.Element {
   const [confirmPassword, setConfirmPassword] = useState<string>('');
   const [error, setError] = useState<string>('');
   const [currentStepIndex, setCurrentStepIndex] = useState<number>(0);
+  const [seedPhrase, setSeedPhrase] = useState<string>('');
+  const { getSeed, changePassword } = useSecretKey();
   const navigate = useNavigate();
-  const dispatch = useDispatch();
-  const { seedPhrase } = useSelector((state: StoreState) => ({
-    ...state.walletState,
-  }));
   const { t } = useTranslation('translation', { keyPrefix: 'CREATE_PASSWORD_SCREEN' });
   const { createWallet } = useWalletReducer();
   const { disableWalletExistsGuard } = useWalletExistsContext();
 
   useEffect(() => {
-    if (!seedPhrase) {
-      navigate('/backup');
-    }
-  }, [seedPhrase]);
+    (async () => {
+      const seed = await getSeed();
+      setSeedPhrase(seed);
+    })();
+  }, []);
```

Comment:
why do we use the `useState` pattern in this screen, and not call `await getSeed()` within `handleConfirmPassword` handler, as in other screens?

Reply:
true, resolved